ui/src/literature/components/LiteratureItem.jsx

```diff
@@ -48,21 +49,27 @@ class LiteratureItem extends Component {
         leftActions={
           <Fragment>
             {arxivId && <ArxivPdfDownloadAction arxivId={arxivId} />}
-            {doi && <DOILinkAction doi={doi} />}
+            {dois && <DOILinkAction dois={dois} />}
             <CiteModalAction recordId={recordId} />
             <EditRecordActionContainer recordId={recordId} />
           </Fragment>
         }
         rightActions={
           <Fragment>
             {citationCount != null && (
-              <EventTracker eventId="Citations:Search">
-                <ListItemAction
-                  iconType="login"
-                  text={`${citationCount} citations`}
-                  link={{ to: `${LITERATURE}?q=refersto:recid:${recordId}` }}
-                />
-              </EventTracker>
+              <ListItemAction>
+                <EventTracker eventId="Citations:Search">
+                  <Link
+                    className="no-transition ant-btn"
```

Comment:
why is this necessary?

Reply:
It was added in the ListItemAction before as well. It makes the citations link not be blue (unlike the other links in the result item) and not have a transition when you click on it.